src/ec/packages/ec-component-accordion/ecl-accordion.html.twig

```diff
@@ -54,23 +54,35 @@
   {% if _items is not empty %}
     {% for _item in _items %}
       <h{{ _item.level|default(3) }} class="ecl-accordion__title">
-        {% include '@ecl-twig/ec-component-button/ecl-button.html.twig' with {
-          label: _item.toggle.label,
-          variant: 'ghost',
-          type: 'button',
-          icon: _item.toggle.icon,
-          icon_position: 'before',
-          extra_classes: 'ecl-accordion__toggle',
-          extra_attributes: [
-            { name: 'aria-controls', value: _item.id },
-            { name: 'data-ecl-accordion-toggle' },
-          ]
-        } only %}
+        <button
```

Comment:
The issue here was to give the a give and attribute specifically for the accordion. I have discussed this with @papegaill and we thought this was the better way since this is also used in the accordion2